src/examples/accordions/stepper/StepperDefault.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Stepper } from '@zendeskgarden/react-accordions';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledContent = styled.div`
+  margin: 0 0 ${p => p.theme.space.sm} 0;
+`;
+
+const Example = () => {
+  const [step, setStep] = useState(0);
+
+  const onNext = () => setStep(step + 1);
+  const onBack = () => setStep(step - 1);
+
+  return (
+    <Row>
+      <Col>
+        <Stepper activeIndex={step}>
+          <Stepper.Step key="step-1">
+            <Stepper.Label>Choose a good location</Stepper.Label>
+            <Stepper.Content>
+              <StyledContent>
+                Your garden&apos;s success depends on its location, so choose a spot that has
+                healthy soil, gets good light, and is easily watered.
+              </StyledContent>
+              <Button onClick={onNext}>Next</Button>
```

Comment:
Seems like there's something wrong with the focusvisible focus state in CodeSandbox. Also wasn't able to navigate to the button using the keyboard on the website 🤔 .

<img width="264" alt="Screen Shot 2020-09-03 at 4 24 41 PM" src="https://user-images.githubusercontent.com/1811365/92183013-06d21d00-ee02-11ea-822e-56585bb003e2.png">